tests/unit/test_plotting/test_plot.py

```diff
@@ -38,23 +37,13 @@ def test_plot2D(self):
 
         _, ax = plt.subplots()
         ax_out = pybamm.plot2D(X, Y, Y, ax=ax, show_plot=False)
-        self.assertEqual(ax_out, ax)
+        assert ax_out == ax
 
-    def test_plot2D_fail(self):
+    def test_plot2_d_fail(self):
```

Comment:
```suggestion
    def test_plot2D_fail(self):
```

Reply:
Still need to figure out whats causing this.

Reply:
If you don't manage to, you can always serve the community and file a bug :)